packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -315,6 +323,11 @@ export async function scanParentDirs(
     cliType = 'npm';
     lockfilePath = npmLockPath;
     lockfileVersion = packageLockJson.lockfileVersion;
+  } else if (bunLockBin) {
+    cliType = 'bun';
+    lockfilePath = bunLockPath;
+    // TODO: read "bun-lockfile-format-v0"
```

Comment:
Lets not forget to make a linear ticket for this

Reply:
We can create a ticket when Bun 2.0 comes out or if the lockfile changes (hopfully that would be semver major but you never know).

I asked here https://github.com/orgs/vercel/discussions/2021#discussioncomment-6968013